packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1358,6 +1361,7 @@ export interface CallWithChatClientState {
     isTeamsMeeting: boolean;
     latestCallErrors: AdapterErrors;
     latestChatErrors: AdapterErrors;
+    mainMeetingCallId?: string;
```

Comment:
Is this needed if its already stored in the call state? Can it instead be accessed instead through the `call` interface already in the CallWithChatClientState

Reply:
I believe the `call` in CallWithChatClientState refers to the active call. Maybe we align this to be `breakoutRoomOrigin` to match the other handler?

Reply:
My mistake. The `breakoutRoomOrigin ` does not need to be exposed in CallWithChatClientState. I originially had it in the adapter state before moving it to the stateful layer. Removed.